features/storage/quota.feature

```diff
@@ -22,7 +22,14 @@ Feature: ResourceQuata for storage
       | ["metadata"]["annotations"]["volume.alpha.kubernetes.io/storage-class"] | foo           |
       | ["spec"]["resources"]["requests"]["storage"]                            | 3Gi           |
     Then the step should succeed
-    And the "pvc-#{ cb.i }" PVC becomes :bound
+    """
+    And I run the steps 3 times:
```

Comment:
You can put pod creation steps after the pvc creation steps.  No need to loop again here.